src/styles/index.scss

```diff
@@ -3 +1,209 @@
+  padding: 0 50px;
+  background-color: $white-color;
+
+  .logo-link {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__list {
+    list-style: none;
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: $font-size-small;
+    color: $text-color;
+    line-height: 60px;
+    display: block;
+    text-align: center;
+    position: relative;
+
+    @include transition(color);
+
+    &:hover {
+      color: $main-color;
+    }
+
+    &.is-active {
+      color: $main-color;
+
+      &::after {
+        content: '';
+        position: absolute;
+        width: 100%;
+        background-color: $main-color;
+        height: 4px;
+        border-radius: $border-radius;
+        left: 0;
+        bottom: 0;
+      }
+    }
+  }
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: $gap-vertical $gap-horizontal;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.card {
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  display: flex;
+  flex-direction: column;
+  padding: $card-padding;
+
+  @include transition(transform, background-color, color);
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: $font-size-small;
+    font-weight: 500;
+    line-height: 150%;
+    color: $text-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    font-size: $font-size-small;
+    line-height: 18px;
+    margin-bottom: 16px;
+    color: #333;
+
+    &-name {
+      font-weight: 400;
+    }
+
+    &-number {
+      font-weight: 700;
+      font-size: $font-size-large;
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    margin: 0 auto;
+    border-radius: $border-radius;
+    background-color: $main-color;
+    color: $white-color;
+    text-decoration: none;
+    line-height: 114%;
+    font-size: 14px;
+    font-weight: 700;
+
+    @include transition(background-color, color);
+
+    &:hover {
+      background-color: $white-color;
+      border: 1px solid $main-color;
+      color: $main-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: $hover-color;
+    }
+  }
```

Comment:
The hover effect on the card scales it by 1.2, which might cause layout shifts and affect user experience. Consider using a smaller scale factor or other visual effects to enhance the hover interaction without causing significant layout changes.